pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -121,8 +122,8 @@ backendStdenv.mkDerivation rec {
     (placeholder "lib")
     (placeholder "out")
     "${placeholder "out"}/nvvm"
-    # Is it not handled by autoPatchelf automatically?
-    "${lib.getLib backendStdenv.cc.cc}/lib64"
+    # NOTE: use the same libstdc++ as the rest of nixpkgs, not from backendStdenv
+    "${lib.getLib stdenv.cc.cc}/lib64"
```

Comment:
I thought we just did a bunch of work refactoring so that we _could_ use `backendStdenv`'s `lib`? why are we reverting?

Reply:
ah ok thanks for explaining... so can we get rid of `backendStdenv` entirely then? IIUC it should no longer be used after this PR?